korgan/claim_exemplar_architecture.py

```
@@ -0,0 +1,176 @@
+
+
+def architecture_issues(case_context: str, research: LegalResearch, draft: ClaimDraft) -> list[str]:
+    context = (case_context or "").lower()
+    reasoning = "\n".join([*draft.facts, *draft.legal_basis, draft.late_interest or ""]).lower()
+    prayer = "\n".join(draft.requests).lower()
+    issues: list[str] = []
+
+    if any(x in context for x in ("неустойк", "пеня", "пени")) and any(x in reasoning for x in ("неустойк", "пеня", "пени")) and not any(x in prayer for x in ("неустойк", "пеня", "пени")):
+        issues.append("Мотивировка обосновывает неустойку/пеню, но требование отсутствует в ПРОШУ СУД.")
+    if ("353" in reasoning or "чужими деньгами" in reasoning) and not ("353" in prayer or "чужими деньгами" in prayer or "неустойк" in prayer):
+        issues.append("Мотивировка содержит самостоятельное денежное требование по статье 353/чужим деньгам, но петитум его не отражает либо мотивировка должна быть удалена как нерелевантная.")
```

Ruff (0.16.2)

[warning] String contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF001)